### DISCUSSION TITLE
Fork to Closed Source Repo

### DISCUSSION THREAD

**Discussion by Vyse777** [apollographql/router#3315] [Q&A] [answered]
Hi there,

Before I do anything, I am curious if it violates any policies/licenses if someone forks this repo into a closed source repo (i.e. Azure DevOps).

For some context: My company has a policy that we are not allowed to pull code from GitHub, and we build the project in a custom Docker container from source so we can run it in a container that is not Debian based. We were hoping we could sync this repo with one we own (and there will be absolutely no code changes/modifications to this project) so we have an internal reference to the codebase.

Sorry if this seems like a weird question, if you have any other follow up questions please let me know! Thank you!

**Comment by chandrikas** [comment, marked as answer]
Hi @Vyse777 

Yes you can fork into a closed source repo so long as you observe the terms and conditions of the ELv2 license described here: https://www.elastic.co/licensing/elastic-license. Please take special note of Limitations and Notices.

**Reply by Vyse777** [reply]
Thank you for the info!
